# Array of traffic light bounding boxes
std_msgs/Header        header
TrafficLightBBox[]     boxes  # List of traffic light bounding boxes